tests/Directory.Build.props

```diff
@@ -14,14 +14,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.core" Version="2.3.0-beta2-build3683" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0-beta2-build1317" />
-    <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="15.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />

```

Comment:
I think we should have been using this instead of `Microsoft.TestPlatform.TestHost`. Anyway, xUnit's "getting started" section uses this package.